codegenerator/cli/templates/static/codegen/src/db/Db.res

```diff
@@ -37,7 +37,6 @@ let allStaticTables: array<Table.table> = [
   TablesStatic.PersistedState.table,
   TablesStatic.EndOfBlockRangeScannedData.table,
   TablesStatic.RawEvents.table,
-  TablesStatic.DynamicContractRegistry.table,
```

Comment:
Is this only affecting hasura visibility? Even so, Could be a breaking change unfortunately

Reply:
It's not, since the table is a part of Entity tables. So this was duplicated here.

Reply:
Oh I see 👍🏼